lib/fs.js

```diff
@@ -1543,6 +1543,9 @@ fs.realpathSync = function realpathSync(p, options) {
       if (linkTarget === null) {
         fs.statSync(base);
         linkTarget = fs.readlinkSync(base);
+        if (linkTarget.startsWith('\\ContainerMappedDirectories')) {
```

Comment:
This seems a bit hacky. Wouldn't this prevent any application from using `\ContainerMappedDirectories`? I imagine an `isWindows` check should be added too.